monkey/common/network/network_range.py

```diff
@@ -170,7 +170,7 @@ def __iter__(self):
             yield self._number_to_ip(self.get_range()[0])
 
     def is_in_range(self, ip_address):
-        return self._ip_address == ip_address
+        return self._ip_address == str(ip_address)
```

Comment:
Why not make `self._ip_address` of type `IPv4Address`?

Reply:
`is_in_range` has calls that passes strings to it (reports call it with strings, because they output strings). We could do a larger refactoring of the report, but I don't want to fix code that shouldn't be there in the first place. I think the solution is to move the report to the UI. For now I just type cast these to make it more fool-proof